src/MyPlot/EventListener.php

```diff
@@ -1,356 +0,0 @@
-<?php
```

Comment:
The Event Listener class is necessary for the plugin to run. What are you doing?

Reply:
I added EventListener in the end.
I needed to renew it, what wasn't that easy on mobile.